lib/errors.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Append the old error message to the new one and keep its stack trace.
+ * Example:
+ *     throw wrap(e, HighLevelError, "This error is more specific");
+ */
+function wrap(
+    old_error,
```

Comment:
We should lint this, but var names should be camelCase

Reply:
https://github.com/matrix-org/matrix-appservice-bridge/pull/115